### ISSUE TITLE
Automatically extend annual allowance every year

### ISSUE THREAD

**Issue by vpp** [timeoff-management/application#87]
As per suggestion from #86 it would be good to have a feature when employees allowance is extended every year.

**Comment by AsimPervez93** [comment]
Could I enquire when this will be ready to roll out?

Would it be possible to automatically extend annual allowance for every year the employee has been with the company? 

i.e. I started in 10 July 2015, on 10 July 2016 my allowance went up by 1 and on 10 July 2017 it will go up by 1 again.

**Comment by vpp** [comment]
Yeah, this ticket is about the functionality you've just described.

TBH, I do not have any particular dates in mind. But your enquiry bumps priority.
